runtimeconfig/google/cloud/runtimeconfig/variable.py

```diff
@@ -139,10 +139,13 @@ def state(self):
 
         :rtype: str
         :returns:
-            If set, one of "UPDATED", "DELETED", or
-            "VARIABLE_STATE_UNSPECIFIED", else ``None``.
+            If set, one of "UPDATED", "DELETED", or defaults to
+            "VARIABLE_STATE_UNSPECIFIED".
         """
-        return self._properties.get('state')
```

Comment:
You can pass the default value as a second argument to `dict.get`, e.g.:

```python
        return self._properties.get('state', STATE_UNSPECIFIED)
```

Reply:
fixed, PTAL